project/dexter-monitor/routes/project.js

```diff
@@ -71,6 +75,68 @@ exports.saveSnapshotSummary = function() {
         })
 };
 
+exports.createProject = function(req, res) {
+
+	Promise.resolve(req.body.project)
+	.then(createDexterServerDatabase)
+	.then(insertDexterMonitorEntry)
+	.then((project) => {
+		startDexterServer(project);
```

Comment:
Actually, we cannot use a promise here, because we're starting a new external process. Therefore, we don't know when dexter-server is "ready". We can only hope that it is ready after some seconds - hence this 5 sec timeout.